docs/upgrade/upgrade-7.17-8.x.asciidoc

```diff
@@ -0,0 +1,155 @@
+
+[float]
+=== Perform 8.x upgrade on Non-Prod (Optional) or Prod Deployment
+
+NOTE: We strongly recommend you perform the following steps on a non-prod test deployment first to address any potential issues before upgrading on a prod deployment. 
+
+. Back up all of your data to a {ref}/snapshots-take-snapshot.html[snapshot] – which is a backup of an index taken from a running cluster. Rollbacks require a snapshot and cause data loss without one.
+
+. We recommend you <<rules-api-export, export all your custom detection rules>> in the event there are issues with the detection engine after the upgrade.
+
+. Upgrade {es}. Refer to {stack-ref}/upgrading-elasticsearch.html[these upgrade instructions]. 
+** If you're using {ecloud}, we recommend upgrades with no downtime. Refer to {cloud}/ec-upgrade-deployment.html[these instructions].  
+** If you're using {ece} (ECE), refer to {ece-ref}/ece-upgrade-deployment.html[these instructions].  
+** If you're using {eck} (ECK), refer to {eck-ref}/k8s-upgrading-stack.html[these instructions]. 
+** If you're upgrading a self-orchestrated deployment, upgrade the data nodes first, tier-by-tier, in this order:
+... Frozen tier
+... Cold tier 
+... Warm tier
+... Hot tier 
+... Any other nodes not in a tier
+... All remaining nodes that are neither master-eligible nor data nodes
+... Master-eligible nodes
+
+. Upgrade {kib}. Refer to {stack-ref}/upgrading-kibana.html[these instructions].
++
+NOTE: If you're using {ecloud} or {ece}, this is already included in the {es} upgrade.
+
+. Validate {es} and {kib} are operating as expected by completing the following checks: 
+.. {es}
+... Check the status of your clusters and ensure they're green. Run the `GET _cat/health` API request. For more information, refer to the {ref}/cat-health.html[cat health API documentation].
+... Check the size of your rolled over indices/shards, which should be less than 50 GB. Run the `GET _cat/shards?s=store:desc` API request. For more information, refer to the {ref}/cat-shards.html[cat shards API documentation].    
+... Ensure the index and search rate are close to what they were prior to upgrading. Go to **Stack Monitoring** -> **ES** -> **Overview**.
++
+TIP: You can also check the index doc count using the {ref}/cat-indices.html[cat index API].
+... Verify the {slm} (SLM) is taking snapshots by {ref}/snapshots-take-snapshot.html#check-slm-history[checking the SLM history]. 
+... Ensure {ml} is up and running. 
+.. {kib} 
+... Ensure that you and your users can successfully log in to {kib} and access desired pages.
+... Check {kibana-ref}/discover.html[Discover] and verify that the index patterns you normally use are available.
+... Verify that your commonly-used {kibana-ref}/dashboard.html[dashboards] are available and working properly.
+... Check your SIEM detection rules, and ensure that the desired set of rules is enabled (rule management) and that enabled rules are running without errors or warnings (rule monitoring).
```

Comment:
Is this item redundant with the similar item below under Validate that Elastic Security is operating OK?